### DISCUSSION TITLE
expand/ collapse rows similiar to react table

### DISCUSSION THREAD

**Discussion by usrbinr** [rstudio/gt#1576] [Ideas]
Apologies if this is repeat,  I tried my best to search but couldn't find an open discussion topic or issue on this.

are there any plans to do collapase / expand rows similiar to the reactable table package?

I think that would be extremely helpful!

**Comment by olivroy** [comment]
In the upcoming (and current dev) gt 0.11.0, you can do:

```r
 mtcars |> gt::gt(groupname_col = "vs") |> gt::opt_interactive()
```

![image](https://github.com/rstudio/gt/assets/52606734/171d51f5-f84e-4891-96d8-d398b7898e4b)


https://github.com/rstudio/gt/issues/1705

**Reply by usrbinr** [reply]
oh it works so well!! thank you. Honestly 100% what my projected needed.

**Reply by olivroy** [reply]
We are still missing the summary rows and grand summary rows integration, but open to contributions #1359
